registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java

```diff
@@ -13,30 +13,11 @@
 import io.mosip.kernel.crypto.jce.core.CryptoCore;
 import io.mosip.kernel.keygenerator.bouncycastle.KeyGenerator;
 import io.mosip.registration.processor.packet.storage.utils.AuthUtil;
-
-import io.mosip.registration.processor.stages.osivalidator.OSIValidator;
-import io.mosip.registration.processor.stages.osivalidator.OSIValidatorStage;
-import io.mosip.registration.processor.stages.osivalidator.UMCValidator;
 import io.mosip.registration.processor.stages.osivalidator.utils.OSIUtils;
 
 @Configuration
 public class OSIConfigBean {
 
-	@Bean
-	public OSIValidator getOSIValidator() {

```

Comment:
Do we need to remove these non stage beans ?

Reply:
These beans are added as part of componentScan in OSIValidatorStage